package-lock.json

```diff
@@ -97,34 +97,144 @@
       }
     },
     "@babel/core": {
-      "version": "7.10.4",
-      "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.10.4.tgz",
-      "integrity": "sha512-3A0tS0HWpy4XujGc7QtOIHTeNwUgWaZc/WuS5YQrfhU67jnVmsD6OGPc1AKHH0LJHQICGncy3+YUjIhVlfDdcA==",
+      "version": "7.12.3",

```

Comment:
I believe we should not upgrade the dependency version. Maybe you need to revert the changes on `package-lock.json` and just remove the  package may not enough.